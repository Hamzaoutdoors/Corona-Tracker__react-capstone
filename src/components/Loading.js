import React from 'react';

const Loading = () => (
  <div className="loader-container">
    <div className="loader" />
  </div>
);

export default Loading;
