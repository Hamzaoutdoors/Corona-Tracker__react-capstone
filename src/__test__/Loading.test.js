import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/configureStore';
import Loading from '../components/Loading';

test('About page renders properly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <BrowserRouter>
          <Loading />
        </BrowserRouter>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
