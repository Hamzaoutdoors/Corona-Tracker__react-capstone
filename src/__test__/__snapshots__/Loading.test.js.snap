// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page renders properly 1`] = `
<div
  className="loader-container"
>
  <div
    className="loader"
  />
</div>
`;
